### PR TITLE
[doc] SID: add RQ-VAE / RQ-KMeans semantic-ID generation docs

### PR DESCRIPTION
**Pull request by WhiteSwan1**
## What

Adds user documentation for the two semantic-ID (SID) generation models and wires them into the Sphinx docs tree.

| File | Change |
| --- | --- |
| `docs/source/models/sid_rqvae.md` | new — `SidRqvae` (RQ-VAE) usage doc |
| `docs/source/models/sid_rqkmeans.md` | new — `SidRqkmeans` (FAISS residual K-Means) usage doc |
| `docs/source/models/sid_model.rst` | new — `语义ID生成` toctree grouping the two docs |
| `docs/images/models/rqvae.png` | new — RQ-VAE architecture diagram |
| `docs/source/index.rst` | register `models/sid_model` under the MODEL section |

## Details

- **sid_rqvae.md** — encoder MLP → multi-level residual vector quantizer →  decoder MLP, STE forward, optional dual-view contrastive path. Documents the  Parquet data format (`item:embedding` array columns), every `sid_rqvae`  config field, the train and SID-generation (`tzrec.predict`) commands, and the  `codes` output, plus the non-contrastive (item-only) variant.
- **sid_rqkmeans.md** — FAISS residual K-Means: CPU-only / single-process,  reservoir sampling during training + a single offline fit in `on_train_end`.  Documents `faiss_kmeans_kwargs`, the `train_config` semantics (optimizers /  epochs are framework formalities for this non-gradient model), and the  `N >= max(codebook)` fit requirement.

### CODE REVIEW

File: docs/source/models/sid_rqkmeans.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+}
+```
+
+- train_config: 训练流程配置。RQKMeans 不做梯度训练, 因此优化器与 epoch 数仅为框架训练循环所需的形式配置
+  - sparse_optimizer / dense_optimizer: 必填 (训练循环要构建优化器), 但模型只有一个 dummy 参数、损失恒为 0, **因此其学习率不影响最终码本**
+  - num_epochs: **设 `1` 即可**; 每个 epoch 只把 embedding 流式写入蓄水池采样 (reservoir), 真正的 FAISS 拟合在训练结束 (`on_train_end`) 时一次性完成, 多跑 epoch 不会迭代/精炼码本
+  - save_checkpoints_steps / save_checkpoints_epochs: **必须项，设 `0` 关闭周期性保存**; 拟合好的码本只随训练结束时的最终 checkpoint 持久化 (周期性保存可能会忽略checkpoint落盘, 故关闭)
+- feature_configs / feature_groups: 同 RQVAE, 但只需主物品 embedding 一组 (`deep`); 其拼接后的总维度即 K-Means 的向量维度
+- sid_rqkmeans: RQKMeans 模型参数
+  - codebook: 每层聚类中心数; **列表长度即残差层数 (= SID 的位数)**; 示例 `[256, 256, 256]` (生产常用 `[8192, 8192, 8192]`); 支持非均匀如 `[256, 512, 1024]` (每层独立拟合一个 faiss.Kmeans)
```
Comment (poson) [inline docs/source/models/sid_rqkmeans.md]: 需要增加一些描述对用户更加友好：
codebook：[256, 512, 1024]，256相当于一级类目的个数，512 相当于二级类目的个数。
总语义id的数量是：256*512*1024 = 134217728 。因此需要使用者根据自己的业务情况来设置语义id的数量。

File: docs/source/models/sid_rqkmeans.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+[sid_generation_item_only_sample_4w.parquet](https://tzrec.oss-accelerate.aliyuncs.com/data/models/sid_generation_item_only_sample_4w.parquet)
+
+### 配置文件
+
+[sid_rq_kmeans.config](https://tzrec.oss-accelerate.aliyuncs.com/config/models/sid_rq_kmeans.config)
+
+### 训练参数
+
+```bash
+OMP_NUM_THREADS=$(nproc) \
```
Comment (poson) [inline docs/source/models/sid_rqkmeans.md:87]: 这里再增加一个读 MaxCompute表的案例？

**Review by tiankongdeguiji — APPROVED** [review]
